plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -508,7 +508,7 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
                         icebergTable.schema().findType(field.sourceId())))
                 .toArray(Type[]::new);
 
-        AppendFiles appendFiles = transaction.newFastAppend();
+        AppendFiles appendFiles = transaction.newAppend();
```

Comment:
What is the behavior when `commit.manifest.min-count-to-merge` is not set?

Reply:
@findepi The default value of `commit.manifest.min-count-to-merge` is 100. `newAppend` will only merge manifests if the size of the manifests is greater than 100.